modules/administration-guide/partials/assembly_understanding-che-workspace-controller.adoc

```diff
@@ -5,6 +5,8 @@
 [id="understanding-{prod-id-short}-workspace-controller_{context}"]
 = Understanding {prod-short} workspace controller
```

Comment:
IBM SG
```
If the subject is a functional overview, begin a heading with words such as Introduction or Overview rather than
contriving a pseudo-task-oriented heading that begins with Understanding, Using, Introducing, or Learning.
```